src/python/WMCore/Services/WMStats/WMStatsReader.py

```diff
@@ -221,12 +220,10 @@ def _getLatestJobInfo(self, keys):
         """
         if len(keys) == 0:
             return []
-        options = {}
-        options["reduce"] = True
-        options["group"] = True
+        options = {"include_docs": True}
+        options["reduce"] = False
         result = self._getCouchView("latestRequest", options, keys)
-        ids = [row['value']['id'] for row in result["rows"]]
-        return ids
+        return result
```

Comment:
for consistency with the other methods, shouldn't we return something like `result['rows']` instead? Or maybe parse it and return only the tuple requestname and agenturl, dropping the None value(?)

Reply:
_getCouchView handling that format. It has to be fed to other function. So format cannot me changed.

Reply:
I don't see _getCouchView changing anything on the output. If it does, then why this function has this line for example
https://github.com/dmwm/WMCore/blob/b25964f7f5eaa57018b414f6273d9220957fa895/src/python/WMCore/Services/WMStats/WMStatsReader.py#L211
?